test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/ConditionalPatchTests.cs

```diff
@@ -117,17 +137,24 @@ public async Task GivenSecondVersionOfResource_WhenPatchingConditionallyWithOneM
             response.Resource.BirthDate = "2020-01-01";
             using FhirResponse<Patient> secondVersion = await _client.UpdateAsync(response.Resource);
 
-            string patchDocument =
-           "[{\"op\":\"replace\",\"path\":\"/gender\",\"value\":\"female\"}, {\"op\":\"remove\",\"path\":\"/address\"}]";
-
-            using FhirResponse<Patient> patchResponse = await _client.ConditionalPatchAsync<Patient>(
+            using FhirResponse<Patient> jsonPatchResponse = await _client.ConditionalJsonPatchAsync<Patient>(
                 "Patient",
                 $"identifier={identifier}",
-                patchDocument,
+                _patchDocumentJson,
                 "2");
-            Assert.Equal(AdministrativeGender.Female, patchResponse.Resource.Gender);
-            Assert.Empty(patchResponse.Resource.Address);
-            Assert.Equal(HttpStatusCode.OK, patchResponse.StatusCode);
+            using FhirResponse<Patient> fhirPatchResponse = await _client.ConditionalFhirPatchAsync<Patient>(
+                "Patient",
+                $"identifier={identifier}",
+                _fhirPatchRequest,
+                "3");
+
+            Assert.Equal(AdministrativeGender.Female, jsonPatchResponse.Resource.Gender);
+            Assert.Empty(jsonPatchResponse.Resource.Address);
+            Assert.Equal(HttpStatusCode.OK, jsonPatchResponse.StatusCode);
+
+            Assert.Equal(AdministrativeGender.Female, fhirPatchResponse.Resource.Gender);
```

Comment:
Same comment as above. When changes are being made there should be separate tests.

Reply:
This one still needs to be split into two tests.

Reply:
I'm going to blame this on yesterday being Monday and point at the cleanup commit I did yesterday to distract you. 😆 

But seriously, missed this one, it's now split into multiple tests. Thanks.